addons/prometheus/0.38.x/prometheus-2.yaml

```diff
@@ -255,8 +255,8 @@ spec:
         alertmanagerSpec:
           resources:
             limits:
-              cpu: 200m
-              memory: 250Mi
+              cpu: 100m
+              memory: 50Mi
```

Comment:
didn't we want to bump these values though?

Reply:
I did it in `prometheus-3.yaml` the PR #298 that bumped them should have created a new revision file (`prometheus-3.yaml`) but instead the previous `prometheus-2.yaml` was overwritten

Reply:
oh that was my miss 👍